docs/modules/deployment/pages/core/getting-started.adoc

```diff
@@ -132,6 +132,36 @@ include::docker/initialize.adoc[]
 endif::[]
 ====
 
+== Encrypting database credentials
+
+It is also possible to store the PostgreSQL credentials in the Secure Credentials Vault.
```

Comment:
```suggestion
It is also possible to store the PostgreSQL credentials in the secure credentials vault.
```
Why is this important? Are there any considerations that users should be aware of if they store PostgreSQL credentials in the SCV?